repo-infra/verify/boilerplate/BUILD

```diff
@@ -0,0 +1,3 @@
+package(default_visibility = ["//visibility:public"])
+
+exports_files(glob(["*.txt"]))
```

Comment:
as far as I understand, glide is for go projects (maybe it can pull in arbitrary repositories). In the world of makefiles we need to build a lot of makefile boilerplate to clone the repo with code generator, build it, then run it to produce the code generated file, and then use it in the context of another make rule. Because we won't want the generated file(s) to pollute the repository tree, we'll want to write them into some separate directory which will make makefiles and dependencies even more complicated. It can be done, but it is tedious.

Bazel has first-class support for all of that - take some random repo, build tool that is inside of it, run it to produce some outputs, and use those outputs to build my target. All quite easy and with all the right guarantees (things get rebuilt when they need to)